tests/raftstore/test_snap.rs

```diff
@@ -213,7 +213,6 @@ fn test_concurrent_snap<T: Simulator>(cluster: &mut Cluster<T>) {
     cluster.must_put(b"k3", b"v3");
     // Now peer 1 can't send snapshot to peer 2
     cluster.sim.wl().add_recv_filter(3, box PauseFirstSnapshotFilter::new());
```

Comment:
We may need to rewrite PauseFirstSnapshotFilter to pause all snapshots until different target region showing up.